tests/e2e/yamcs/network.e2e.spec.js

```diff
@@ -100,12 +101,13 @@ test.describe("Quickstart network requests @yamcs", () => {
         // wait for debounced requests in YAMCS Latest Telemetry Provider to finish
         await new Promise(resolve => setTimeout(resolve, 500));
         filteredRequests = filterNonFetchRequests(networkRequests);
-
+        console.log(filteredRequests);
```

Comment:
if you can remove this, we should. Not a big deal to print diagnostic to screen in tests